geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -692,7 +692,7 @@ See <a href="../../../configuring/cluster_config/gfsh_persist.html">Overview of
 </tr>
 <tr class="odd">
 <td><span class="keyword parmname">\-\-hostname-for-clients</span></td>
-<td>Sets the ip address or host name that this cache server is to listen on for client connections. Setting a specific hostname-for-clients causes server locators to use this value when telling clients how to connect to this cache server. This is useful in the case where the cache server may refer to itself with one hostname, but the clients need to use a different hostname to find the cache server. The value &quot;&quot; causes the bind-address to be given to clients. A null value will be treated the same as the default &quot;&quot;.</td>
+<td>Sets the IP address or host name that this server is to listen on for client connections. Set this value when clients use a different address to connect with the server than the <code>bind-address</code>, as it would in a multi-homed environment. Not specifying this option or setting this option to the empty string (&quot;&quot;) causes the <code>bind-address</code> to be given to clients.</td>
```

Comment:
There's a distinction between how `server-bind-address` and `hostname-for-clients` works: `server-bind-address` affects how the server binds ports, whereas `hostname-for-clients` does not; it only gives a different hostname for clients. This makes it more useful for environments like AWS, or like, behind a NAT or something.

Reply:
I'm not sure exactly how to get that across or if it really matters, but there it is.